dataflow/flex-templates/streaming_beam/streaming_beam_test.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+import multiprocessing as mp
+import os
+import subprocess as sp
+import tempfile
+import time
+
+from google.cloud import bigquery
+from google.cloud import pubsub
+
+PROJECT = os.environ["GCLOUD_PROJECT"]
+DATASET = 'beam_samples'
+TABLE = 'streaming_beam_sql'
+TOPIC = 'messages'
+SUBSCRIPTION = 'messages'
+
+
+def create_topic_path():
+    publisher_client = pubsub.PublisherClient()
+    topic_path = publisher_client.topic_path(PROJECT, TOPIC)
+    try:
+        publisher_client.delete_topic(topic_path)
+    except Exception:
+        pass
+    response = publisher_client.create_topic(topic_path)
+    return response.name
```

Comment:
You can do `yield topic.name` instead and then continue to delete the topic afterwards, that way we can cleanup for the resources created.

Reply:
Done.